# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/bsd-games/postfix-bsd.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./banner/Makefrag.orig	Sun Sep 26 10:44:31 1999
+++ ./banner/Makefrag	Wed Sep 29 07:29:04 1999
@@ -31,6 +31,8 @@
 banner_all:	banner/banner banner/banner.6
 
 banner_install:	banner_all
-	$(INSTALL_BINARY) banner/banner $(INSTALL_PREFIX)$(GAMESDIR)/banner
-	$(HIDE_GAME) banner
-	$(INSTALL_MANUAL) banner/banner.6
+	$(INSTALL_BINARY) banner/banner $(INSTALL_PREFIX)$(GAMESDIR)/banner-bsd
+	$(HIDE_GAME) banner-bsd
+	ln -f banner/banner.6 banner/banner-bsd.6
+	$(INSTALL_MANUAL) banner/banner-bsd.6
+	rm -f banner/banner-bsd.6
--- ./factor/Makefrag.orig	Sun Sep 26 10:48:55 1999
+++ ./factor/Makefrag	Wed Sep 29 07:30:34 1999
@@ -34,6 +34,8 @@
 factor_all:	factor/factor factor/factor.6
 
 factor_install:	factor_all
-	$(INSTALL_BINARY) factor/factor $(INSTALL_PREFIX)$(GAMESDIR)/factor
-	$(HIDE_GAME) factor
-	$(INSTALL_MANUAL) factor/factor.6
+	$(INSTALL_BINARY) factor/factor $(INSTALL_PREFIX)$(GAMESDIR)/factor-bsd
+	$(HIDE_GAME) factor-bsd
+	ln -f factor/factor.6 factor/factor-bsd.6
+	$(INSTALL_MANUAL) factor/factor-bsd.6
+	rm -f factor/factor-bsd.6
--- ./worm/Makefrag.orig	Sun Sep 26 10:57:24 1999
+++ ./worm/Makefrag	Wed Sep 29 07:29:04 1999
@@ -31,6 +31,8 @@
 worm_all:	worm/worm worm/worm.6
 
 worm_install:	worm_all
-	$(INSTALL_BINARY) worm/worm $(INSTALL_PREFIX)$(GAMESDIR)/worm
-	$(HIDE_GAME) worm
-	$(INSTALL_MANUAL) worm/worm.6
+	$(INSTALL_BINARY) worm/worm $(INSTALL_PREFIX)$(GAMESDIR)/worm-bsd
+	$(HIDE_GAME) worm-bsd
+	ln -f worm/worm.6 worm/worm-bsd.6
+	$(INSTALL_MANUAL) worm/worm-bsd.6
+	rm -f worm/worm-bsd.6
